build.psm1

```diff
@@ -1202,13 +1216,14 @@ function Start-PSPackage {
 
 
 function New-UnixPackage {
-    [CmdletBinding()]param(
+    [CmdletBinding()]
+    param(
         [Parameter(Mandatory)]
         [ValidateSet("deb", "osxpkg", "rpm")]
         [string]$Type,
 
         [Parameter(Mandatory)]
-        [string]$Source,
```

Comment:
Renamed this parameter to PackageSourcePath for consistency across packing functions